wal/src/manager.rs

```diff
@@ -207,15 +223,23 @@ impl ReadBoundary {
 
 #[derive(Debug, Clone)]
 pub struct ReadRequest {
-    /// Region id of the wal to read
-    pub region_id: RegionId,
+    /// WalLocation of the wal to read
+    pub wal_location: WalLocation,
     // TODO(yingwen): Or just rename to ReadBound?
     /// Start bound
     pub start: ReadBoundary,
     /// End bound
     pub end: ReadBoundary,
 }
 
+#[derive(Debug, Clone)]
+pub struct ScanRequest {
+    /// WalLocation of the wal to read
+    pub wal_location: WalLocation,
```

Comment:
Shouldn't `scan` read the data of the whole region? Otherwise, it seems no essential difference between the `read_batch` and `scan`.

Reply:
Done.